<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastro_usuarios</class>
 <widget class="QDialog" name="cadastro_usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_cadastro_usuario">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1071</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Cadastro Usuário</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1041</width>
       <height>410</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Nome Completo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_nome"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Username</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_username"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Senha</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_senha"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>CEP</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txt_cad_usu_cep">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Endereço</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_endereco"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Telefone</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_fone"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Tipo usuário</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_cad_usu_tipo_usuario"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Sexo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_cad_usu_sexo"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Email</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_email">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Email Alternativo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_email_alt">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Data de Nascimento</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_data_nasc">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Pais</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_cad_usu_pais">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="btn_cadastro_usuario">
            <property name="text">
             <string>Cadastrar Usuário</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_cancelar">
            <property name="text">
             <string>Cancelar (Limpar Tela)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_36">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Acesso:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Consultar e editar Usuário</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>331</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtrar Usuário</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_37">
            <item>
             <widget class="QRadioButton" name="rb_filtrar_username">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_filtrar_nome">
              <property name="text">
               <string>Nome</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_filtrar_sexo">
              <property name="text">
               <string>Sexo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_39">
            <item>
             <widget class="QRadioButton" name="rb_filtrar_cep">
              <property name="text">
               <string>CEP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_filtrar_fone">
              <property name="text">
               <string>Fone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_filtrar_email">
              <property name="text">
               <string>Email</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QLineEdit" name="txt_filtrar_usuario"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_filtrar_usuario">
              <property name="text">
               <string>Filtrar Usuário</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>348</x>
       <y>11</y>
       <width>711</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_38">
      <item>
       <widget class="QTableWidget" name="tableWidget_consulta_usuario"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btn_editar_usuario">
          <property name="text">
           <string>Editar Usuário</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_atualizar_usuario">
          <property name="text">
           <string>Atualizar Usuário</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_excluir_usuario">
          <property name="text">
           <string>Excluir Usuário</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Acessos disponíveis do Sistema:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidget_total"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_17">
              <item>
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>Acessos do Usuário:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidget_atual_usu"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="btn_add_acesso">
              <property name="text">
               <string>Adicionar Acesso</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_excluir_acesso">
              <property name="text">
               <string>Excluir Acesso</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>330</width>
       <height>330</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_22">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_username"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Nome Completo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_nome"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_24">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Senha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_senha"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_31">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_email">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_29">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Tipo usuário</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_atual_usu_tipo_usuario"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_30">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Sexo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_atual_usu_sexo"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_26">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>CEP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_cep">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_32">
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Email Alternativo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_email_alt">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_25">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Endereço</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_endereco"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_33">
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Data de Nascimento</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_data_nasc">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Telefone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_fone"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_34">
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Pais</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_atual_usu_pais">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
